src/Sylius/Component/Core/Provider/LastYearSalesSummaryByPostgresProvider.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Provider;
+
+use Doctrine\ORM\EntityRepository;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\OrderPaymentStates;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+
+final class LastYearSalesSummaryByPostgresProvider implements LastYearSalesSummaryProviderInterface
+{
+    /** @var OrderRepositoryInterface|EntityRepository */
+    private $orderRepository;
+
+    public function __construct(OrderRepositoryInterface $orderRepository)
+    {
+        $this->orderRepository = $orderRepository;
+    }
+
+    /** @psalm-suppress PossiblyUndefinedMethod */
+    public function getSalesSummary(ChannelInterface $channel, \DateTimeInterface $startDate, \DateTimeInterface $endDate): array
+    {
+        $queryBuilder = $this->orderRepository->createQueryBuilder('o')
```

Comment:
I would prefer to make this implementation called default and return empty data, as we are not able to test it right now (or to set up PostgreSQL build on Travis)